<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="KAPPO-NOTE" name="runner.TestRunner" tests="15" failures="3" timestamp="2021-02-17T09:56:37 BRT" time="5.259" errors="0">
  <testcase name="runScenario" time="0.055" classname="runner.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.065" classname="runner.TestRunner">
    <failure type="java.lang.AssertionError" message="expected:&lt;400&gt; but was:&lt;201&gt;">
      <![CDATA[java.lang.AssertionError: expected:<400> but was:<201>
at org.junit.Assert.fail(Assert.java:89)
at org.junit.Assert.failNotEquals(Assert.java:835)
at org.junit.Assert.assertEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:633)
at steps.StepDefinitions.validarStatusCode(StepDefinitions.java:66)
at ✽.Eu devo verificar que o HTTP StatusCode retornado é 400(file:///home/mkapobianco/Projects/TestesAutomatizadosBrasilPrev/src/test/java/features/CadastrarClientes.feature:61)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.040" classname="runner.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.051" classname="runner.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.060" classname="runner.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.040" classname="runner.TestRunner">
    <failure type="java.lang.AssertionError" message="expected:&lt;400&gt; but was:&lt;500&gt;">
      <![CDATA[java.lang.AssertionError: expected:<400> but was:<500>
at org.junit.Assert.fail(Assert.java:89)
at org.junit.Assert.failNotEquals(Assert.java:835)
at org.junit.Assert.assertEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:633)
at steps.StepDefinitions.validarStatusCode(StepDefinitions.java:66)
at ✽.Eu devo verificar que o HTTP StatusCode retornado é 400(file:///home/mkapobianco/Projects/TestesAutomatizadosBrasilPrev/src/test/java/features/CadastrarClientes.feature:48)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.032" classname="runner.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.071" classname="runner.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.037" classname="runner.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.035" classname="runner.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.030" classname="runner.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="4.386" classname="runner.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.203" classname="runner.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.136" classname="runner.TestRunner">
    <failure type="java.lang.AssertionError" message="expected:&lt;400&gt; but was:&lt;500&gt;">
      <![CDATA[java.lang.AssertionError: expected:<400> but was:<500>
at org.junit.Assert.fail(Assert.java:89)
at org.junit.Assert.failNotEquals(Assert.java:835)
at org.junit.Assert.assertEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:633)
at steps.StepDefinitions.validarStatusCode(StepDefinitions.java:66)
at ✽.Eu devo verificar que o HTTP StatusCode retornado é 400(file:///home/mkapobianco/Projects/TestesAutomatizadosBrasilPrev/src/test/java/features/CadastrarClientes.feature:41)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.018" classname="runner.TestRunner"/>
  <system-out/>
</testsuite> <!-- runner.TestRunner -->
